pkg/kv/txn_coord_sender_test.go

```diff
@@ -467,6 +467,8 @@ func TestTxnCoordSenderHeartbeat(t *testing.T) {
 
 // getTxn fetches the requested key and returns the transaction info.
 func getTxn(ctx context.Context, txn *client.Txn) (*roachpb.Transaction, *roachpb.Error) {
+	// Note that this HeartbeatTxnRequest doesn't set the Now field, so it doesn't
+	// update the txn. It acts simply as an easy way to read the txn record.
```

Comment:
Can't you use a Push in Query mode? I'd rather do that and add a check against the zero value for Now.